src/PaymentSuite/RedsysBundle/phpunit.xml.dist

```diff
@@ -27,8 +27,5 @@
             </exclude>
         </whitelist>
     </filter>
-    <php>
```

Comment:
Are you sure ? it's a recommandation from Symfony documentation => http://symfony.com/doc/current/book/testing.html#your-first-functional-test